pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "chip-gateway-hl7-listener"
+version = "1.0.0"
+description = "HL7 listener service for CHIP gateway appliance"
+authors = ["sajal-g <sajalgoyal@gmail.com>"]
+
+packages = [
+    { include = "hl7_listener", from = "src/main/py"},    
+]
+
+[[tool.poetry.source]]
+name = "development-pypi"
+url = "https://coverahealth.jfrog.io/artifactory/api/pypi/development-pypi/simple"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+hl7 = "^0.4.5"
+asyncio-nats-client = "^0.11.5"
```

Comment:
Nitpick: it would warm my cockles if these and dev deps were sorted alphabetically (but keep `python` at the top)

Reply:
Done